config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     containerImage: quay.io/konveyor/oadp-operator:latest
     createdAt: "2020-09-08T12:21:00Z"
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
-      Data Protection Applications on the OpenShift platform.
+      Data Protection Applications on the OpenShift platform. Fake edit :)
```

Comment:
```suggestion
      Data Protection Applications on the OpenShift platform.
```

Reply:
Thanks, was using this to test if it also works with CI ehheeh